lib/Frontend/ParseableInterfaceModuleLoader.cpp

```diff
@@ -1066,9 +1066,12 @@ class ParseableInterfaceModuleLoaderImpl {
   }
 
   bool isInResourceDir(StringRef path) {
-    StringRef resourceDir = ctx.SearchPathOpts.RuntimeLibraryPath;
-    if (resourceDir.empty()) return false;
-    return path.startswith(resourceDir);
+    for (auto &RuntimeLibraryPath : ctx.SearchPathOpts.RuntimeLibraryPaths) {
+      if (path.startswith(RuntimeLibraryPath)) {
```

Comment:
This should just be using `RuntimeResourcePath` to begin with. The system libraries are *not* in the compiler resource directory.

Reply:
Right you are.